grype/pkg/cpe_provider.go

```diff
@@ -39,61 +42,3 @@ func getCPEReader(userInput string) (r io.Reader, ctx Context, err error) {
 	}
 	return nil, ctx, errDoesNotProvide
 }
-
-func decodeCPEsFromReader(reader io.Reader, ctx Context) ([]Package, Context, *sbom.SBOM, error) {
-	scanner := bufio.NewScanner(reader)
-	var packages []Package
-	var syftPkgs []pkg.Package
-
-	for scanner.Scan() {
-		rawLine := scanner.Text()
-		p, syftPkg, err := cpeToPackage(rawLine)
-		if err != nil {
-			return nil, Context{}, nil, err
-		}
-
-		if p != nil {
-			packages = append(packages, *p)
-		}
-		if syftPkg != nil {
-			syftPkgs = append(syftPkgs, *syftPkg)
-		}
-	}
-
-	if err := scanner.Err(); err != nil {
-		return nil, Context{}, nil, err
-	}
-
-	s := &sbom.SBOM{
-		Artifacts: sbom.Artifacts{
-			Packages: pkg.NewCollection(syftPkgs...),
-		},
-	}
-
-	return packages, ctx, s, nil
-}
-
-func cpeToPackage(rawLine string) (*Package, *pkg.Package, error) {
-	c, err := cpe.New(rawLine, "")
-	if err != nil {
-		return nil, nil, fmt.Errorf("unable to decode cpe %q: %w", rawLine, err)
-	}
-
-	syftPkg := pkg.Package{
-		Name:    c.Attributes.Product,
-		Version: c.Attributes.Version,
-		CPEs:    []cpe.CPE{c},
-		Type:    internal.CPETargetSoftwareToPackageType(c.Attributes.TargetSW),
```

Comment:
Should the [CPETargetSoftwareToPackageType](https://github.com/anchore/grype/blob/b1706ca14e88428cddb30ef72a09867b9df09851/grype/internal/cpe_target_software_to_pkg_type.go#L11) get removed now, too since it's in syft?